### PR TITLE
Bug - Handle External Images Not Found Better

### PR DESCRIPTION
**Pull request by boothy060590**
### Description
- Instead of failing out and throwing InvalidImageException when an invalid external img source is present to us and therefore failing to export in the API. We now just return a paragraph element containing to broken URL in question instead. This will allow the export to continue to run and not fail a whole project because of one invalid file. 
- Added test for this case

Fixes # (issue)

### Checklist:

- [x] I have run `composer run-script check --timeout=0` and no errors were reported
- [x] The new code is covered by unit tests (check build/coverage for coverage report)
- [ ] I have updated the documentation to describe the changes

### CODE REVIEW

**Comment by boothy060590** [comment]
[[Regular] Unable to export projects to PDF or Word](https://trello.com/c/i05mEXTN/1998-regular-unable-to-export-projects-to-pdf-or-word)

File: tests/PhpWord/Shared/HtmlTest.php [hunk 1]
```diff
@@ -605,6 +606,21 @@ public function testCouldNotLoadImage()
         Html::addHtml($section, $html, false, true);
     }
 
+    public function testExternalImageSourceNotFound()
+    {
+        $src = 'https://www.bridgewatersavings.com/assets/1442845771-FDIC.png'; // returns 404
```
Comment (petemcfarlane) [inline tests/PhpWord/Shared/HtmlTest.php:611]: 😄 I love the url chosen

File: tests/PhpWord/Shared/HtmlTest.php [hunk 2]
```diff
@@ -605,6 +606,21 @@ public function testCouldNotLoadImage()
...
+
+        $phpWord = new \PhpOffice\PhpWord\PhpWord();
+        $section = $phpWord->addSection();
+        $html = '<p><img src="' . $src . '" width="150" height="200" style="float: right;"/></p>';
+        Html::addHtml($section, $html, false, true);
+
+        $doc = TestHelperDOCX::getDocument($phpWord, 'Word2007');
+
+        $baseXpath = '/w:document/w:body/w:p/w:r';
+        $this->assertTrue($doc->elementExists($baseXpath . '/w:t'));
```
Comment (petemcfarlane) [inline tests/PhpWord/Shared/HtmlTest.php:621]: 🟢 Could we check if we see the src image instead?

File: src/PhpWord/Shared/Html.php [hunk 3]
```diff
@@ -869,7 +870,11 @@ protected static function parseImage($node, $element)
         }
 
         if (is_file($src)) {
-            $newElement = $element->addImage($src, $style);
+            try {
+                $newElement = $element->addImage($src, $style);
+            } catch (InvalidImageException $ex) {
+                $newElement = $element->addText($src);
```
Comment (petemcfarlane) [inline src/PhpWord/Shared/Html.php]: I guess just returning the string src is the bare minimum, perhaps we could prefix it like 
```suggestion
                $newElement = $element->addText("Error: Could not load image: {$src}");
```

**Review by petemcfarlane — APPROVED** [review]
speedy! 🏃

File: tests/PhpWord/Shared/HtmlTest.php (hunk 2 above)
Comment (boothy060590) [reply tests/PhpWord/Shared/HtmlTest.php:621]: sure

File: src/PhpWord/Shared/Html.php (hunk 3 above)
Comment (boothy060590) [reply src/PhpWord/Shared/Html.php]: Good shout, will do

**Pull request merged**